### PR TITLE
Removal of Apptentive::Duration, to_s for Version

### PR DESCRIPTION
**Pull request by msaffitz**

### CODE REVIEW

**Comment by msaffitz** [comment]
Tests succeeded. Ready for code review.

File: lib/types/version.rb [hunk 1]
```diff
@@ -13,13 +13,18 @@ def initialize(code)
...
+    alias_method :inspect, :to_s
+
     def <=>(that)
       code <=> (that.respond_to?(:code) ? that.code : that)
     end
 
     # (E)JSON serialization
     def as_json(*)
-      { _type: "version", version: @code.join(".") }.as_json
+      { _type: "version", version: code.join(".") }.as_json
```
Comment (stim371) [inline lib/types/version.rb]: should we just call `inspect` or `to_s` in the version value?

**Comment by stim371** [comment]
otherwise :+1:

File: lib/types/version.rb (hunk 1 above)
Comment (msaffitz) [inline lib/types/version.rb]: Yeah, that's cleaner.  Thanks

**Comment by msaffitz** [comment]
Tests succeeded. Ready for code review.

**Pull request merged**